contrib/cli/requirements.txt

```diff
@@ -1,4 +1,4 @@
-black==22.3.0        # Linting / code style
+black[jupyter]==22.3.0 # Linting / code style
```

Comment:
Initially I also update the version of black to `v23` but that introduced a bunch of formatting changes to `.py` files so I reverted.